functions/AutoRemediateS3-002.js

```diff
@@ -0,0 +1,78 @@
+"use strict";
+
+const config = require('./config');
+const AWS = require("aws-sdk");
+
+const CCRuleCode = 'S3-002'
+const CCRuleName = 'BucketPublicReadAcpAccess'
+const allUsersURI = 'http://acs.amazonaws.com/groups/global/AllUsers'
+const readAcpPermission = "READ_ACP"
+
+module.exports = {
+
+  remediateAllUsers: function (thisGrant, newAcl) {
```

Comment:
Can these functions be extracted into a common library - there is commonality between this and other S-xxx rules.